client/components/helpCentre/contactUs/contactUsForm.tsx

```diff
@@ -180,6 +152,34 @@ export const ContactUsForm = (props: ContactUsFormProps) => {
 		return isFormInValidState;
 	};
 
+	useEffect(() => {
+		if (window.grecaptcha) {
+			renderReCaptcha();
+		} else {
+			const script = document.createElement('script');
+			script.setAttribute(
+				'src',
+				'https://www.google.com/recaptcha/api.js?onload=v2ReCaptchaOnLoadCallback&render=explicit',
+			);
+			// tslint:disable-next-line:no-object-mutation
+			window.v2ReCaptchaOnLoadCallback = renderReCaptcha;
+			document.head.appendChild(script);
+		}
+	}, []);
+
+	useEffect(() => {
+		if (!formValidationState.fileAttachment.isValid) {
+			validateForm();
+		}
+	});
```

Comment:
Why have you removed the dependencies here?

Reply:
Hey, that's because adding the `validateForm` function to the dependency array causes a re-render loop. One solution would be to wrap the `validateForm` function in a useCallback hook ... or like here I have removed the dependency array entirely and let the `validateForm` function be called on every page update, seeing as it's not an expensive function to run and the page doesn't re-render that often this seemed like the easiest solution

Reply:
That makes sense, thanks!